### DISCUSSION TITLE
Stopped working for me as of today. Anyone else see the same?

### DISCUSSION THREAD

**Discussion by python-qwsa** [UltimaHoarder/UltimaScraper#1849] [Q&A]
I think my cookie config is fine as my method has been working for a while.

All of the sudden I am getting this:

Auth (V1) Attempt 1/10
Access denied.

Anyone else?

**Comment by roweger** [comment]
You probably either need to update the cookie, or upgrade to the latest release on the fork:

https://github.com/DIGITALCRIMINALS/OnlyFans/releases
